pandas/core/nanops.py

```diff
@@ -629,7 +654,7 @@ def _get_counts_nanvar(
     mask: Optional[np.ndarray],
     axis: Optional[int],
     ddof: int,
-    dtype=float,
+    dtype: Dtype = float,
```

Comment:
@simonjayhawkins my read of Dtype is that `float` shouldn't be included.  Is this not-failing because np.dtype resolves to Any?

Reply:
yes, definition of Dtype alias is being updated in #29046 to allow this.

Reply:
So keep the ```Dtype``` typing here? or remove it?

Reply:
OK to keep IMO